src/main/java/entities/Car.java

```diff
@@ -0,0 +1,158 @@
+package entities;
+
+import java.io.Serializable;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+
+
+@Entity
+@NamedQuery(name = "CAR.deleteAllRows", query = "DELETE from CAR")
```

Comment:
not sure if uppercase works, but approved :D

Reply:
I'll change it

Reply:
The build is failing anyway

Reply:
Should be ok now